script.js

```diff
@@ -0,0 +1,253 @@
+                    <i class="fas fa-ellipsis-v"></i>
+                </button>
+                <div class="dropdown-menu p-2 flex-column" aria-labelledby="dropdownMenuItem1" id="${task.id}">
+                    ${!task.isDone ? `<button type="button" class="btn btn-success w-100">Complete</button>
+    <button type="button" class="btn btn-info w-100 my-2" data-toggle="modal" data-target="#exampleModal">Edit</button>`
+        : `<button type="button" class="btn btn-success w-100 mb-2">Uncomplete</button>`}
+                    <button type="button" class="btn btn-danger w-100">Delete</button>
+                </div>
+            </div>`;
+    if (sortUp) {
+        task.isDone ? completedTasksBlock.append(taskLayout) : currentTasksBlock.append(taskLayout);
+    } else {
+        task.isDone ? completedTasksBlock.prepend(taskLayout) : currentTasksBlock.prepend(taskLayout);
+    }
+}
+
+function renderApp() {
+    currentTasksBlock.innerHTML = null;
+    completedTasksBlock.innerHTML = null;
+    allTasks.map(task => addTaskLayout(task));
+    const tasksCount = getTasksCount();
+    currentTasksTitle.textContent = `ToDo (${tasksCount.current})`;
+    completedTasksTitle.textContent = `Completed (${tasksCount.completed})`;
+}
+
+function removeTask(taskId) {
+    const copyTasks = allTasks.filter(task => task.id !== taskId);
+    updateLocalStorage(LocalStorageKeys.tasks, copyTasks);
+    allTasks = copyTasks;
+    renderApp();
+}
+
+function changeTaskStatus(taskId) {
+    const copyTasks = allTasks.map(task => task.id === taskId ? {...task, isDone: !task.isDone} : task);
+    updateLocalStorage(LocalStorageKeys.tasks, copyTasks);
+    allTasks = copyTasks;
+    renderApp();
+}
+
+function showEditModal(taskId) {
+    const task = allTasks.find(task => task.id === taskId);
+    modalLabelText.textContent = KeysVariables.text.editTask;
+    submitButton.textContent = KeysVariables.text.save;
+    inputTitle.value = task.title;
+    inputText.value = task.text;
+    form.elements[task.priority].checked = true;
+    editTaskId = taskId;
+    modalEditMode = true;
+}
+
+
+function tasksBlockListener({target}) {
+    const dropDownMenu = target.closest('.dropdown-menu');
+    if (dropDownMenu) {
+        const taskId = dropDownMenu.getAttribute('id');
+        if (target.classList.contains(KeysVariables.buttons.editBtn)) {
+            showEditModal(taskId);
+        } else if (target.classList.contains(KeysVariables.buttons.changeStatusBtn)) {
+            changeTaskStatus(taskId);
+        } else {
+            removeTask(taskId);
+        }
+    }
+}
+
+currentTasksBlock.addEventListener('click', tasksBlockListener);
+completedTasksBlock.addEventListener('click', tasksBlockListener);
+
+function closeModal() {
+    modalLabelText.textContent = KeysVariables.text.addTask;
+    submitButton.textContent = KeysVariables.text.addTask;
+    form.querySelectorAll('.form-check-input')[1].checked = true;
```

Comment:
`form.querySelectorAll('.form-check-input')[1]` - if you need exactly this selector than create a variable with clear name for it

Reply:
```suggestion
    form.elements[PRIORITIES.medium].checked = true;
```
Changed to this expression. Is it ok?